src/app/calendar/_actions/modifyReservation.tsx

```diff
@@ -0,0 +1,38 @@
+"use server";
+
+import { getToken } from "@/auth/getToken";
+import { MODIFY_RESERVATION_URL } from "@/server-endpoints/reservations";
+
+export async function modifyReservation(formData: FormData) {
+  const token = await getToken();
+  if (!token) {
+    throw new Error("Not authenticated");
+  }
+  const id = formData.get("id") as string;
+  const title = formData.get("title") as string;
+  const description = formData.get("description") as string;
+  const type = formData.get("type") as string;
+
+  const body = {
+    title,
+    description,
+    type,
+  };
+  console.log(body);
```

Comment:
here

Reply:
it helps me while developing application to control if everything still works properly